### ISSUE TITLE
Add undocumented components

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds-site#808]
There are a number of components that exist in USWDS that are not documented on the website or not shown as standalone components. Let's add them or document them explicitly:

- `usa-skipnav` #1030 
- `usa-media-block`#1031
- `usa-hero` #1028 
- `usa-checklist` #1029 
- `responsive embed container`  #1101 

**Other**
- graphic list needs further discussion if this should be a separate component or part of media block https://github.com/orgs/uswds/projects/1#card-50342748

[Inventory and crosswalk](https://docs.google.com/spreadsheets/d/1rM1JkbL-RGwE3DRXhvD7fYpogZ-yYHLh9Wt8DBAQc4k/edit#gid=0)

**Comment by jAigret-Bix** [comment]
Separate issues have been created and linked in description for each needed component page. Work will be tracked in individual issues and this issue can be closed.
